specification.md

```diff
@@ -795,6 +795,44 @@ Path? getStderrPath()
 
 Set/get the paths to the standard stream files.
 
+<a name="jobspec-setprelaunch"></a>
+<a name="jobspec-setpreexec"></a>
+<a name="jobspec-setpostexec"></a>
+<a name="jobspec-setpostlaunch"></a>
+```java
+void setPreLaunch(Path script)
+Path? getPreLaunch()
+void setPreExec(Path script)
+Path? getPreExec()
+void setPostExec(Path script)
+Path? getPostExec()
```

Comment:
I seem to vaguely recall a discussion in which we may have agreed that pre- and post- exec can be done by a wrapper script, which does not require API support.

Reply:
In that case we should remove the `pre-exec` signatures.  We should then also mention `JPSI_RANK` which we intended to export to the task which supports `pre_exec` scripts.  That might be a  different PR though as we have discussed some other env vars too IIRC and might want to tackle this in one go.

Reply:
I think you are looking at an outdated quote and @SteVwonder has already removed those signatures (although perhaps not the anchors).

Reply:
You are right, only the anchors are left.

Reply:
Just pushed a fixup commit with those anchors removed.